### ISSUE TITLE
Include full history on .kc3 files

### ISSUE THREAD

**Issue by McDohl** [KC3Kai/KC3Kai#491]
I tried export to export full data then clear my resources and consumables from database and try to import the data again and it's gone QwQ :disappointed: . 

So the question is, export data contains what? Is it intended to not include resources and consumables?

**Comment by LostLogia4** [comment]
iirc it's a JSON entry record includes among other things, your <i>API links</i>. I think it only have stuffs that is in your local storage. Everything else is stored in sqlite database or something.

**Comment by dragonjet** [comment]
yep, it only contains HQ profile, api link, ships, equipment, stats, fleet compos, maps.

it does not include history based on actions, such as construction, sortie, etc.

at least not yet :rocket:

**Comment by LostLogia4** [comment]
So where are these located?
Is there a way to back them up?

**Comment by dragonjet** [comment]
the history are saved onto the browser's `IndexedDB` in contrast to basic info being on `localStorage`. You can actually see them by going to strategy room, opening the devtools > resources > expand IndexedDB > expand KC3 > explore the tables

We can pull them and include onto the JSON file actually, but that export feature was actually done in a rush, so it didn't have all the expected features.

If there is no other issue pointing to this, we can keep this open to track the inclusion of history on the export

**Comment by LostLogia4** [comment]
Thanks for explaining just when I was about to inquire about this. I suppose we'll just keep ths issue open until then.
